source/install.sh

```diff
@@ -4,7 +4,7 @@ SOURCE_PATH=$(dirname "$SCRIPT")
 
```

Comment:
Partly because I didn't want to change everything in one go, and partly because instructions / help to install locally should still be available at some level in the end